docs/Deprecating-Disabling-and-Removing-Casks.md

```diff
@@ -0,0 +1,110 @@
+
+## Deprecation
+
+If a user attempts to install a deprecated cask, they will be shown a warning message but the install will proceed.
+
+A cask should be deprecated to indicate to users that the cask should not be used and will be disabled in the future. Deprecated casks should continue to be maintained by the Homebrew maintainers if they continue to be installable. If this is not possible, they should be immediately disabled.
+
+The most common reasons for deprecation are when the upstream project is deprecated, unmaintained, or archived.
+
+Casks should only be be deprecated if at least one of the following are true:
+
+- the cask does not install on any of our supported macOS versions
+- the cask has outstanding CVEs
+
+To deprecate a cask, add a `deprecate!` call. This call should include a deprecation date (in the ISO 8601 format) and a deprecation reason:
+
+```ruby
+deprecate! date: "YYYY-MM-DD", because: :reason
+```
+
+The `date` parameter should be set to the date that the project or version became (or will become) deprecated. If there is no clear date but the cask needs to be deprecated, use today's date. If the `date` parameter is set to a date in the future, the cask will not become deprecated until that date. This can be useful if the upstream developers have indicated a date when the project or version will stop being supported.
+
+The `because` parameter can be a preset reason (using a symbol) or a custom reason. See the [Deprecate and Disable Reasons](#deprecate-and-disable-reasons) section below for more details about the `because` parameter.
+
+## Disabling
+
+If a user attempts to install a disabled cask, they will be shown an error message and the install will fail.
+
+A cask should be disabled to indicate to users that the cask cannot be used and will be removed in the future. Disabled cask are those that could not be installed successfully any longer.
+
+The most common reasons for disabling a cask are:
+
+- it cannot be installed
+- it has been deprecated in Homebrew for at least six months
+- the upstream URL has been removed or uses archive.org
+
+Popular casks (i.e. have more than 999 [analytics installs in the last 30 days](https://formulae.brew.sh/analytics/cask-install/30d/)) should be deprecated for at least six months even if e.g. they do not install on all macOS versions before being disabled.
+
+Unpopular casks (i.e. have fewer than 1000 [analytics installs in the last 30 days](https://formulae.brew.sh/analytics/cask-install/30d/)) can be deprecated immediately for any of the reasons above e.g. they cannot be installed.
+
+**Note: disabled casks in `homebrew/cask` will be automatically removed one year after their disable date.**
+
+Unpopular casks (i.e. have fewer than 1000 [analytics installs in the last 30 days](https://formulae.brew.sh/analytics/cask-install/30d/)) can be manually removed three months after their disable date.
```

Comment:
```suggestion
Casks having fewer than 1000 [analytics installs in the last 30 days](https://formulae.brew.sh/analytics/cask-install/30d/) can be manually removed three months after their disable date.
```

Reply:
I'd actually sooner change the i.e. to an e.g. as I don't think popularity is best judged entirely on a number in our analytics.